lms/djangoapps/discussion_api/views.py

```diff
@@ -158,7 +159,9 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
             "unanswered" for question threads with no marked answer. Only one
             can be selected.
 
-        The topic_id, text_search, and following parameters are mutually
+        * inline_discussion_id: ID of an inline discussion for any module
```

Comment:
Thanks @muzaffaryousaf. @nasthagiri I don't have a lot of context right now but if we are using "inline discussion" throughout, then this string looks fine. My only uncertainty is why we need to say "any module". I would be happy to do a pass through the entire file and take a look at all docstrings. What's the timeline for this PR?